### ISSUE TITLE
Elastic Search node should have option to return full search data

### ISSUE THREAD

**Issue by skylord123** [julien-pal/node-red-contrib-elasticsearch#2]
I used this package to populate elasticsearch but now I want to create a search controller that returns results and information about how long the search took, etc. 

Can we get a new config option on the Elastic Search node that enables the full response to be returned in the payload?

EDIT: It would also be nice if we got support for the 'q' option under search instead of just the body. It would make running simple searches easier.

Thanks.

**Comment by julien-pal** [comment]
Hi, 
I've published a new version (1.1.3) with the full response option.
Right now I don't have the time to add the support of the q option so it would be nice to open another issue for this subject so I can close this one.

**Comment by skylord123** [comment]
@julien-pal I created #3 for this. I will close this now, thanks.

**Comment by skylord123** [comment]
This actually isn't working.

https://github.com/julien-pal/node-red-contrib-elasticsearch/blob/master/elasticsearch/elasticsearch-search.js#L44-L45

`searchConfig.fullResponse` should be `config.fullResponse`.

I also would like this to work with scrolling so I did an update that appends all the responses to an array and returns that. This way the user has access to all the raw responses and has the ability to use scrolling with full responses (I saw you made scrolling not work with this):
(elasticsearch-search.js)
https://gist.github.com/skylord123/c72451cb94f3a36f35162d41730cdaf5

I set the raw responses to `msg.es_responses` as an array. The documentation for this node should be updated to say this gets returned in the output if Full Response is true.
